sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/aggregate/CountIfSuite.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.expressions.aggregate
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.expressions._
+import org.apache.spark.sql.types._
+
+class CountIfSuite extends SparkFunSuite {
+  val input: Attribute = AttributeReference("input", BooleanType, nullable = false)()
+  val evaluator: DeclarativeAggregateEvaluator =
+    DeclarativeAggregateEvaluator(CountIf(input), Seq(input))
+
+  test("empty") {
+    val result: InternalRow = evaluator.initialize()
+    assert(result === InternalRow(0L))
+  }
+
+  test("update") {
+    // update with non-nulls
+    val result1: InternalRow = evaluator.update(
+      InternalRow(true),
+      InternalRow(false),
+      InternalRow(true)
+    )
```

Comment:
Nit: I would move ')' to the previous line, based on the example used in https://github.com/databricks/scala-style-guide given for the section "For method and class constructor invocations, use 2 space indentation for its parameters and put each in each line when the parameters don't fit in two lines."

Reply:
Thanks for information! I’ll adjust :)

Reply:
Okay, I’ll adjust it :)